e2e/tests/nightly/admin/lock-workspace.spec.ts

```diff
@@ -65,9 +65,9 @@ describe('Workspace Admin lock-workspace', () => {
     expect(aboutLockReason).toEqual(reasonText);
     const aboutLockedIcon = aboutPage.getAboutLockedWorkspaceIcon();
     expect(aboutLockedIcon).toBeTruthy();
-    // verify DATA & ANALYSIS tabs are disabled
-    // await aboutPage.openDataPage({ waitPageChange: false });
-    // await aboutPage.openAnalysisPage({ waitPageChange: false });
+    // verify DATA & ANALYSIS tabs are inactive for locked workspace
+    expect(await aboutPage.getTabsAttribute(page, Tabs.Analysis, 'aria-selected')).toEqual('false');
+    expect(await aboutPage.getTabsAttribute(page, Tabs.Data, 'aria-selected')).toEqual('false');
```

Comment:
`Tab` has `isSelected()` which you can use for this check.

Reply:
ok! will try. Thanks!